Motorcycle.js

```diff
@@ -1,6 +1,20 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle(make, model, year, color, passengers) {
```

Comment:
Here, you forgot your seats parameter, which you'll need to include in your ```Car.call()``` function as well.